plugins/pod-logs.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha1
+kind: Plugin
+metadata:
+  name: pod-logs
+spec:
+  platforms:
+  - head: https://github.com/danisla/kubefunc/archive/v1.0.1.zip
+    sha256: c548f94d8efd1fb8aa35a54b7e5cdb64ae012447122d9704f3cee4163bbbe9c3
+    uri: https://github.com/danisla/kubefunc/archive/v1.0.1.zip
+    files:
+    - from: "/*/pod-logs/*"
+      to: "."
+    selector:
+      matchExpressions:
+      - {key: os, operator: In, values: [darwin, linux]}
+  version: "v1.0.1"
+  shortDescription: Get logs from a list of pods
```

Comment:
the description doesn't make it clear how's this different than kubectl-logs.
have you checked out mtail? does it differentiate enough?

Reply:
This is very different than mtail. It presents a list to choose from. 

You should try these plugins first to gauge how they work and differ.

Reply:
Makes sense, we should get this in. Do you foresee a more original name for this or are you happy to use this name (hint: this will impede discoverability for this plugin via search engines)